src/ui/components/Preset.tsx

```diff
@@ -36,18 +60,30 @@ export default function Preset({ preset }: { preset: Preset }) {
 	return (
 		<View>
 			<View>
-				<FormTitle title={preset.name} icon={<SummarizedIconRow
-				iconWrapperStyle={styles.wrapper}
-				items={preset.authorIds.map((id) => UserStore.getUser(id))}
-				max={5}
-				offsetAmount={-8}
-				overflowComponent={OverflowCircle}
-				overflowStyle={styles.wrapper}
-				style={{
-					height: 16,
-				}}
-				renderItem={renderAvatar}
-			/>} />
+				<FormTitle
+					title={preset.name}
+					icon={
+						<SummarizedIconRow
+							iconWrapperStyle={styles.wrapper}
+							items={preset.authorIds.map((id) => {
+								const user = UserStore.getUser(id);
+
+								// Hopefully next time the user will be fetched
+								if (!user) UserUtils.getUser(id);
```

Comment:
you could make a react hook that takes a set of ids and rerenders after they're fetched